chia/rpc/wallet_rpc_api.py

```diff
@@ -999,6 +1001,15 @@ async def send_notification(self, request) -> EndpointResult:
         await self.service.wallet_state_manager.add_pending_transaction(tx)
         return {"tx": tx.to_json_dict_convenience(self.service.config)}
 
+    async def sign_message(self, request) -> EndpointResult:
+        puzzle_hash: bytes32 = decode_puzzle_hash(request["address"])
+        hex_message = request["message"]
+        if hex_message.startswith("0x") or hex_message.startswith("0X"):
+            hex_message = hex_message[2:]
+        message = bytes.fromhex(hex_message)
+        pubkey, signature = await self.service.wallet_state_manager.main_wallet.sign_message(message, puzzle_hash)
+        return {"success": True, "pubkey": str(pubkey), "signature": str(signature)}
```

Comment:
I think `bytes(pubkey).hex()` is better. It's not clear what `str` does.

Reply:
They are same

Reply:
I know, I wrote that code. However, one is more clear than the other.